### PR TITLE
Fix incorrect cluster InitializePhase type

### PR DESCRIPTION
**Pull request by zyfjeff**
Description: Fix incorrect cluster InitializePhase type debug message. At the time of initialization of the cluster, all the InitializePhase type of cluster output were secondary
Risk Level: low
Testing: N/A
Docs Changes: N/A

Signed-off-by: tianqian.zyf <tianqian.zyf@alibaba-inc.com>

### CODE REVIEW

File: source/common/upstream/upstream_impl.cc [hunk 1]
```diff
@@ -917,7 +917,8 @@ void ClusterImplBase::onPreInitComplete() {
   }
   initialization_started_ = true;
 
-  ENVOY_LOG(debug, "initializing secondary cluster {} completed", info()->name());
+  ENVOY_LOG(debug, "initializing {} cluster {} completed",
+            initializePhase() == InitializePhase::Primary ? "Primary" : "Secondary" info()->name());
```
Comment (dio) [inline source/common/upstream/upstream_impl.cc]:
```suggestion
            initializePhase() == InitializePhase::Primary ? "Primary" : "Secondary", info()->name());
```

Comment (zyfjeff) [reply source/common/upstream/upstream_impl.cc]: done

**Comment by jmarantz** [comment]
@snowp for senior maintainer approval

**Comment by zyfjeff** [comment]
Is there anything else this PR needs to be reviewed?

**Comment by jmarantz** [comment]
No more approvals needed, but it looks like CI isn't completely done yet, so I can't merge yet.

**Pull request merged**